Assets/MRTK/Providers/WindowsVoiceInput/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "com.microsoft.mixedreality.toolkit.windowsvoice",

```

Comment:
Can this go more generic? It might be good to just have a `Windows` package?

Reply:
maybe. right now that is all that is in this folder. might be confusing though in that we are not providing anything else.

Reply:
done